src/Symfony/Bundle/WebProfilerBundle/Twig/WebProfilerExtension.php

```diff
@@ -114,10 +115,15 @@ public function getName()
 
     private static function escape(Environment $env, string $s): string
     {
+        // Twig 3.9
         if (method_exists(EscaperExtension::class, 'escape')) {
             return EscaperExtension::escape($env, $s);
         }
 
+        if (class_exists(EscaperRuntime::class) && null !== $escaperExtension = $env->getRuntime(EscaperRuntime::class)) {
```

Comment:
Let's move this above the previous check and add a comment about the version

Reply:
done